packages/js/test/plugins/candyMachineModule/findCandyMachinesByPublicKeyField.test.ts

```diff
@@ -66,3 +67,60 @@ test('[candyMachineModule] find all candy machines by authority', async (t) => {
     );
   });
 });
+
+test('[candyMachineModule] find all candy machines correctly parses token mints', async (t) => {
```

Comment:
This is a great test thanks!

Could we just make the following adjustment to also make sure the collection PDAs are being fetched and parsed properly:

Could the first two candy machines each have their own collection set using the `collection` parameter? You may create them using the helper method `createCollectionNft`.

Then we can ensure they all have the right collection on the results. Feel free to test each candy machine one by one instead of doing a loop to make this easier.

Reply:
@lorisleiva done!